ftw/workspace/profiles/contact/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<metadata>
+  <dependencies>
+    <dependency>profile-ftw.workspace:default</dependency>
```

Comment:
I'd probably make the `contact`-profile an extension profile. I mean it may not install `ftw.workspace:default` but the user should install both.
This would allow to later install the `:contact`-profile without resetting customizations on top of `:default`-options.
Just a thought :wink: